e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,22 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T1797 Compact view: Show single image thumbnail', () => {
+        const filename = 'image-small-height.png';
+
+        // Make a post with some text and a single image
+        cy.get('div[id="centerChannelFooter"]').find('input[id="fileUploadInput"]').attachFile(filename);
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        cy.get('div[class="file__image"]').last().within(() => {
+            // The name of the image appears on a new line
```

Comment:
How could I ensure the file name isn't bolded?

Reply:
The normal font-weight we're using is 400; 600 for bold. Therefore, we can assert css value of it:
```javascript
cy.contains('div', filename).should('be.visible').and('have.css', 'font-weight', '400');
```